packages/tupaia-web/src/features/DashboardItem/EnlargedDashboardItem.tsx

```diff
@@ -179,19 +177,20 @@ export const EnlargedDashboardItem = () => {
     setUrlSearchParams(urlSearchParams);
   };
 
-  const titleText = `${currentDashboardItem?.name}, ${
-    currentDashboardItem?.entityHeader || activeDashboard?.entityName
+  const titleText = `${currentDashboardItem?.config?.name}, ${
+    // @ts-ignore - entityHeader is all lowercase in the types config
+    currentDashboardItem?.config?.entityHeader || entityName
   }`;
 
-  const { type } = currentDashboardItem || {};
+  const type = currentDashboardItem?.config?.type;
 
   return (
     <Modal isOpen onClose={handleCloseModal}>
       <Wrapper $hasBigData={reportData?.data?.length > 20 || type === 'matrix'}>
         <Container>
           <TitleWrapper>
             <BackLink parentDashboardItem={parentDashboardItem} />
-            {currentDashboardItem?.name && <Title>{titleText}</Title>}
+            {currentDashboardItem?.config?.name && <Title>{titleText}</Title>}
```

Comment:
It might be nice to destructure `config` from `currentDashboardItem` above so that we don't need to repeat ourselves as much?